daprdocs/content/en/operations/components/pluggable-components/pluggable-components-discovery.md

```diff
@@ -0,0 +1,229 @@
+---
+type: docs
+title: "How-To: Discover a pluggable component"
+linkTitle: "How To: Discover a pluggable component"
+weight: 4500
+description: "Learn how to help Dapr discover your pluggable component"
+---
+
+[uds]: https://en.wikipedia.org/wiki/Unix_domain_socket
+
+## Service Discovery Process
+
+Pluggable, [gRPC-based](https://grpc.io/) components are typically run as containers or processes that need to communicate with the Dapr main process via [Unix Domain Sockets][uds]. They are automatically discovered and registered in runtime by Dapr using the following steps:
+
+1. The Component listens to an [Unix Domain Socket][uds] placed on the shared volume.
+2. The Dapr runtime lists all [Unix Domain Socket][uds] in the shared volume.
+3. The Dapr runtime connects with the socket and uses gRPC reflection to discover all services that such component implements.
```

Comment:
Services is a confusing term here, since services are used in place for applications. Can this be something like this instead
```suggestion
3. The Dapr runtime connects with each socket and uses gRPC reflection to discover all met methods from a given interface that the component implements.
```